Entity/UserManager.php

```diff
@@ -146,4 +148,28 @@ public function getPager(array $criteria, $page, $limit = 10, array $sort = arra
 
         return $pager;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function loadUserByUsername($username): UserInterface
+    {
+        return $this->findUserByUsername($username);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function refreshUser(SecurityUserInterface $user): UserInterface
```

Comment:
The interface states that `SecurityUserInterface` must be returned (PHP Doc only because SF 3 still support PHP < 7).

IMO to avoid problems when SF 3 will add typehints, we should typehint to `SecurityUserInterface` now.
WDYT @greg0ire?

Reply:
I don't think you can add parameters typehint that the interface does not have. Please check on 3v4l

Reply:
@Matan Same as `loadUserByUsername()`:
> throws UnsupportedUserException if the account is not supported

Reference: https://github.com/symfony/security-core/blob/2.3/User/UserProviderInterface.php#L64

Reply:
@greg0ire https://github.com/sonata-project/SonataEasyExtendsBundle/pull/125#discussion_r136245915

Reply:
@greg0ire Please make sure we both talk about the **return** typehint, not the parameter typehint.

Reply:
We're not, I was talking about parameters typehint. For `return` type hint, I think it is fine, let me check.

Reply:
It is: https://3v4l.org/iA1Mk